### PR TITLE
Integrate old offer presenter into new price calculator page

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Subj. Small change to reuse existing Offer Presenter. Should be replaced by new variant later

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4464?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4464** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4464?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4461** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4461?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4457** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4457?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#4459](https://github.com/HedvigInsurance/racoon/pull/4459) <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4459?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#4458** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4458?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4456** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4456?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4455** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4455?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/PriceCalculatorPage/PriceCalculatorPage.tsx [hunk 1]
```diff
@@ -15,18 +15,13 @@ type Props = {
 // TODO: Convert to vanilla styles when we get to look and feel part
 export function PriceCalculatorPage({ locale }: Props) {
   return (
-    <div
-      className={xStack({})}
-      style={{ maxHeight: '100vh', overflow: 'auto', backgroundColor: 'white', gap: 'initial' }}
-    >
+    <div className={xStack({})} style={{ backgroundColor: 'white', gap: 'initial' }}>
```
Comment (gustaveen) [inline apps/store/src/components/PriceCalculatorPage/PriceCalculatorPage.tsx:18]: Does `xStack` have an initial gap value?

**Review by gustaveen — APPROVED** [review]
👍

File: apps/store/src/components/PriceCalculatorPage/PriceCalculatorPage.tsx (hunk 1 above)
Comment (alebedev) [reply apps/store/src/components/PriceCalculatorPage/PriceCalculatorPage.tsx:18]: Yes, it has some default I wanted to reset

**Pull request merged**